### PR TITLE
Aws tests (at least 2 +1 is required)

### PR DESCRIPTION
**Pull request by doktoric**
This PR is for be able to run AWS tests for you PRs.
here isa Readme about how you can do this: https://cloudera.atlassian.net/wiki/spaces/ENG/pages/186024002/Cloudbreak+Pull+Request+builds

### CODE REVIEW

**Comment by doktoric** [comment]
retest

**Comment by doktoric** [comment]
retest this please

**Comment by doktoric** [comment]
**at least 2 +1 is required !**

File: environment/src/main/resources/schema/app/20190904070547_CB-3270_save_private_subnet_creation_flag.sql [hunk 1]
```diff
@@ -8,4 +8,4 @@ ALTER TABLE environment_network DROP COLUMN IF EXISTS subnetids;
 -- SQL to undo the change goes here.
 
 ALTER TABLE environment_network DROP COLUMN IF EXISTS privateSubnetCreation;
-ALTER TABLE environment_network ADD COLUMN IF NOT EXISTS subnetids TEXT NOT NULL;
+ALTER TABLE environment_network ADD COLUMN IF NOT EXISTS subnetids TEXT;
```
Comment (lnardai) [inline environment/src/main/resources/schema/app/20190904070547_CB-3270_save_private_subnet_creation_flag.sql]: is this intended?

Comment (doktoric) [reply environment/src/main/resources/schema/app/20190904070547_CB-3270_save_private_subnet_creation_flag.sql]: this is a mistake in the revert script

**Pull request merged**